front/src/styles/scss/applications/stdcm/_results.scss

```
@@ -75,13 +84,15 @@
             white-space: nowrap;
 
             .check-circle {
+              display: flex;
+              align-items: center;
+              margin-left: 8px;
               color: rgb(60, 202, 128);
-              padding-left: 8px;
             }
           }
 
           &.selected {
-            .creation-date {
+            .simulation-metadata {
               color: rgb(121, 118, 113);
```

Not coming from your work but 

```suggestion
              color: var(--grey50);
```